src/components/views/rooms/RoomSublist.tsx

```diff
@@ -125,7 +125,7 @@ export default class RoomSublist extends React.Component<IProps, IState> {
     private notificationState: ListNotificationState;
 
     private slidingSyncMode: boolean;
-    private slidingSyncInit: Promise<any>;
+    private slidingSyncInit: Promise<any>; // we don't care about the resulting list, hence 'any'
```

Comment:
Should be `<unknown>` then as then TS will not allow you to operate on the value, unlike `any` where it'll allow anything blindly

```suggestion
    private slidingSyncInit: Promise<unknown>; // we don't care about the resulting list
```